### DISCUSSION TITLE
Exclude parameters from weight decay

### DISCUSSION THREAD

**Discussion by rohan-mehta-1024** [PyTorchLightning/lightning-flash#1461] [Data / pipelines]
I am wondering how to exclude parameters from weight decay when using LARS? Do I have to manually annotate the parameters somehow? I have looked at the source code for the LARS optimizer but am still confused.
